src/Modules/Data/Commands.Data.Test/GetPowerBITableTests.cs

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.PowerBI.Commands.Common.Test;
+using Microsoft.PowerBI.Commands.Profile.Test;
+using Microsoft.PowerBI.Common.Abstractions;
+using Microsoft.PowerBI.Common.Api;
+using Microsoft.PowerBI.Common.Api.Datasets;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System.Linq;
+
+namespace Microsoft.PowerBI.Commands.Data.Test
+{
+    [TestClass]
+    public class GetPowerBITableTests
```

Comment:
Consider adding unit test cases. See GetPowerBIWorkspaceTests.cs for examples. You don't have to add tests for every case for now. However, it's important to add some unit tests since interactive tests won't run with builds triggered by pull requests.

Reply:
I just added question above and now I understand that we should add DI version, too.

Reply:
That's right. Thanks for adding unit tests.